index.js

```diff
@@ -41,7 +42,8 @@ const strings = {
     pause: 'Pause',
     rotate: '180° Rotieren',
     translate: 'Verschieben',
-    connected: 'Dieses Tonnetz ist nicht verbunden, und enthält nicht alle Noten.'
+    connected: 'Dieses Tonnetz ist nicht verbunden, und enthält nicht alle Noten.',
+    notes: ['A', 'Bb', 'B', 'C', 'Db', 'D', 'Eb', 'E', 'F', 'Gb', 'G', 'Ab']
```

Comment:
It actually turns out that in the German version, `Bb` should be `B` and `B` should be `H`

Reply:
Not familiar with German but I've changed it as requested.